CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Master
+
+**⚠️ If you are using android, please make sure to follow the migration guide [here](https://github.com/Instabug/Instabug-React-Native/blob/master/README.md#updating-to-version-863)**
+* Adds initializing android sdk from using `Instabug.start` instead of inside the application class
```

Comment:
```suggestion
* You can now initialize Instabug on Android by calling `Instabug.start` from Javascript, instead of calling the builder method from the application class.
```